ceph_deploy/tests/util.py

```diff
@@ -24,5 +26,12 @@ class Empty(object):
     instantiation.
     """
     def __init__(self, **kw):
-        for k, v in kw.items():
+        for k, v in list(kw.items()):
             setattr(self, k, v)
+
+
+def assert_too_few_arguments(err):
```

Comment:
I am surprised for this addition, would you explain this as well? py.test (the test runner we use) does a great job at managing assertions and comparisons